code/lib/cli/src/sandbox-templates.ts

```diff
@@ -100,6 +103,20 @@ const baseTemplates = {
       builder: '@storybook/builder-webpack5',
     },
     skipTasks: ['e2e-tests-dev', 'bench'],
+    modifications: {
+      mainConfig: (config) => {
+        const stories = config.getFieldValue<Array<StoriesEntry>>(['stories']);
+        return {
+          stories: stories?.map((s) => {
+            if (typeof s === 'string') {
+              return s.replace('js|jsx|mjs|ts|tsx', 'js|jsx|mjs');
```

Comment:
This seems a bit brittle because the order of things might change, or the number of elements might change.

Can you try something like this instead? `s.replace(/\|(tsx?|ts)\b|\b(tsx?|ts)\|/g, '')`

I tested with different formats and it cleaned them correctly